miniconda/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM debian:7.4
+FROM ubuntu:14.04
 
 MAINTAINER Travis Swicegood
 
-RUN apt-get update && apt-get install -y wget bzip2
+RUN apt-get update
+RUN apt-get install -y wget bzip2
 RUN echo 'export PATH=/opt/anaconda/bin:$PATH' > /etc/profile.d/conda.sh
-RUN wget --quiet http://repo.continuum.io/miniconda/Miniconda3-3.6.0-Linux-x86_64.sh && \
-    /bin/bash /Miniconda3-3.6.0-Linux-x86_64.sh -b -p /opt/anaconda && \
-    rm Miniconda3-3.6.0-Linux-x86_64.sh
```

Comment:
This was done for a reason.  As noted in ed48242c9151dbd64b6d5ea7ab475fd9c13942f5, the switch and organization of steps like this is roughly a 30% reduction in size.  The containers should be optimized for size.  Each `RUN` statement increases the size in a way that can not be retrieved.  The container does not need a copy of the `Miniconda-*.sh` file on disk, but without this combination it's stored forever in the image history.

This is bad for Miniconda, and devastating to the size of the Anaconda images.